### ISSUE TITLE
auto configuration of spacy with anaconda

### ISSUE THREAD

**Issue by amatsuo** [quanteda/spacyr#96]
After a bit of search, it seems possible and safe to configure `spacy` automatically in an environment with anaconda. 

A new function will be implemented (say `conda_spacy_config(py_version, languages)`)

Steps will be:

1. set up a condaenv (probably name `spacyr`)
1. install spacy and language models in the environment 
1. write `.Rprofile` if the previous steps are successful

This is even desirable as we can control the version of python `spacy` will be installed in and the environment is independent of the system python.

**Comment by SPLOpenData** [comment]
I am using a new install of anaconda and spacyr within rstudio. Sometimes spacyr works fine - e.g. I can call spacy_parse(txt) and get a result. However, I keep getting the Rstudio "R session has aborted" bomb. Is this related, or do I have a different problem? Could this be related to having to versions of language model 'en' installed. I have one in ~/anaconda3/bin/python and one  in  ~/anaconda3/bin/python3

**Comment by amatsuo** [comment]
@SPLOpenData 

Could you provide the  `sessionInfo()` output?

Do you know which python you are using for spacyr? Also, it would be helpful if you give us the python and spacy versions.

**Comment by SPLOpenData** [comment]
Thanks for your help. Here is the version of conda, spaCy, and the sessionInfo(), below. My Anaconda version comes with python 3.6. I have tried setting ask=TRUE in spacy_initialize selecting both 1 and 2 in separate attempts to make it work.  The value “txt” in the below code is a large character (1.8 Mb) in case that is of any use.

$ conda --version

conda 4.4.10



$ pip show spacy

Name: spacy

Version: 0.101.0

Summary: Industrial-strength NLP

Home-page: https://spacy.io

Author: Matthew Honnibal

Author-email: matt@spacy.io<mailto:matt@spacy.io>

License: MIT

Location: /Users/theuser/anaconda3/lib/python3.6/site-packages

Requires:


> setwd("~/Desktop/")
> df<-read.csv("data_splitting.txt", sep ="\t")
> spd<-subset(df, !grepl('SPD|SFD', df$department_name))
> txt<-spd$request_summary
> txt<-paste(txt, collapse = " ")
> rm(df)
> library('spacyr')
> ## Loading required package: spacyr
> spacy_initialize(ask =TRUE)
Finding a python executable with spacy installed...
spaCy (language model: en) is installed in more than one python
1: /Users/theuser/anaconda3/bin/python
2: /Users/theuser/anaconda3/bin/python3
Please select python: 1
spacyr will use: /Users/theuser/anaconda3/bin/python
successfully initialized (spaCy Version: 0.101.0, language model: en)
> sessionInfo()
R version 3.4.3 (2017-11-30)
Platform: x86_64-apple-darwin15.6.0 (64-bit)
Running under: macOS High Sierra 10.13.3

Matrix products: default
BLAS: /System/Library/Frameworks/Accelerate.framework/Versions/A/Frameworks/vecLib.framework/Versions/A/libBLAS.dylib
LAPACK: /Library/Frameworks/R.framework/Versions/3.4/Resources/lib/libRlapack.dylib

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base

other attached packages:
[1] spacyr_0.9.6

loaded via a namespace (and not attached):
[1] compiler_3.4.3      tools_3.4.3         yaml_2.1.17         Rcpp_0.12.15
[5] reticulate_1.5      data.table_1.10.4-3 jsonlite_1.5

**Comment by amatsuo** [comment]
@kbenoit 
I created a few functions for the implementation of this issue. Could you install the branch `issue-96` and tell me what you think? 

```r
library(spacyr)
spacy_condaenv_setup()
spacy_initialize()
## a conda enviroment 'spacy_condaenv' has been set up. spacyr uses this environment
## successfully initialized (spaCy Version: 2.0.9, language model: en)
```

**Comment by amatsuo** [comment]
Implemented in #98
